Modulefile

```diff
@@ -10,3 +10,4 @@ project_page 'https://github.com/puppetlabs/puppetlabs-debbuilder.git'
 ## Add dependencies, if any:
 dependency 'ploperations/git', '>= 0.0.1'
 dependency 'ploperations/gpg', '>= 0.0.1'
+dependancy 'puppetlabs/stdlib', '>= 0.0.1'
```

Comment:
The minimum version of `stdlib` that includes `ensure_packages` is 3.2.0. Can the version constraint here reflect that (`>= 3.2.0`)?